cocos/core/utils/jsb-utils.ts

```diff
@@ -110,26 +110,25 @@ export function updateChildrenForDeserialize (node: Node) {
     if (!node) {
         return;
     }
+
     // @ts-expect-error: jsb related codes.
-    node._setChildren(node._children);
+    const children = node._children;
+    if (!children) {
+        return;
```

Comment:
Fix incorrect animation state of https://github.com/cocos-creator/example-marionette demo .
node._children is probably an `undefined` value.